src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -19,6 +19,9 @@
 import static java.util.stream.Collectors.joining;
 import static java.util.stream.Collectors.toList;
 import static org.opendatakit.briefcase.export.ExportForms.buildCustomConfPrefix;
+import static org.opendatakit.briefcase.export.ExportOutcome.ALL_EXPORTED;
```

Comment:
This is one of the special cases where I’d argue for the *. We’re using all three.

Reply:
Yup. That would improve the code...